lms/resources/_js_config/__init__.py

```diff
@@ -512,6 +523,18 @@ def _hypothesis_client(self):
             ]
         }
 
+    def forward_lti_parameters(self):
```

Comment:
We call this now in a few places, use a helper function.

It would probably need some docs in the final version as this not trivial to understand what's going on here.